include/deal.II/base/mpi.h

```diff
@@ -2093,6 +2112,72 @@ namespace Utilities
 
 
 
+    template <typename T>
+    T
+    scatter(const MPI_Comm &      comm,
+            const std::vector<T> &objects_to_send,
```

Comment:
I was not sure if we should add an assert to check whether `objects_to_send.size()==n_procs` on `root_process`.

Reply:
I add an assert.